src/components/common/animations/SaveButton/SaveButton.js

```diff
@@ -55,7 +55,7 @@ class SaveButton extends React.Component {
   handleAnimationFinish = () => {
     const { onFinish, loading } = this.props
     const { animStep } = this.state
```

Comment:
on Mobile crash because not find animStep.
maybe with this: 
`constructor(props){
super(props)
this.state = { animStep: 0}
}`

![Screen Shot 2020-03-19 at 12 19 18 PM](https://user-images.githubusercontent.com/59056508/77084060-2f459680-69dd-11ea-8505-d0648ce5d832.png)